features/Skylighting/Shaders/Skylighting/Skylighting.hlsli

```diff
@@ -1,11 +1,13 @@
+#include "Common/Spherical Harmonics/SphericalHarmonics.hlsli"
 
 Texture2D<unorm float> OcclusionMapSampler : register(t29);
+Texture2D<float4> SkylightingTexture : register(t30);
 
 cbuffer SkylightingData : register(b8)
 {
 	row_major float4x4 OcclusionViewProj;
+	float4 EyePosition;
```

Comment:
Not used?

Reply:
ShadowDirection below not used either, only used in Compute version

Reply:
EyePosition should be used when I get around to improving water skylighting.